### DISCUSSION TITLE
dockerfile image for nx-go?

### DISCUSSION THREAD

**Discussion by xonio-x** [nx-go/nx-go#144] [General]
hi guys, any example to write dockerfile for nx-go

`FROM node:23-alpine3.20 `

if i create dockerfile with only node image tidy not working so i need go image for that, but i dont think so go image include node

```
FROM node:23-alpine3.20 AS build

WORKDIR /usr/src/app

COPY --chown=node:node package*.json ./
COPY --chown=node:node . .

ENV GO_VERSION=1.23.0

RUN wget https://golang.org/dl/go${GO_VERSION}.linux-amd64.tar.gz && \
    tar -C /usr/local -xzf go${GO_VERSION}.linux-amd64.tar.gz && \
    rm go${GO_VERSION}.linux-amd64.tar.gz

ENV PATH="/usr/local/go/bin:${PATH}"

RUN npm ci
RUN npx nx run-many -t tidy
RUN npx nx run-many -t build

FROM golang:1.22.6-alpine3.20 AS main
WORKDIR /app

ENV TZ=Asia/Jakarta

COPY --from=build /usr/src/app/dist/apps/atreus main

ENTRYPOINT ["/app/main"]
```

that my current dockerfile

**Comment by utarwyn** [comment]
Hello @xonio-x! Thank you for using the plugin 🔥 
In fact, there don't seem to be any official images with both Node.js and Golang. However, it is possible to have both by installing Node.js in a container based on the Golang image or vice versa, as you've started to do.

What is the exact error you have with this Dockerfile?
